src/lib/components/customer/event/HeroSection.svelte

```
@@ -0,0 +1,114 @@
+        class="inline-flex items-center gap-5 rounded-2xl border border-slate-200 bg-slate-50 px-8 py-4"
+      >
+        {#each stats as stat, i}
+          <div class="text-center">
+            <div class="block text-xl font-black text-purple-700 sm:text-2xl">{stat.number}</div>
+            <div class="text-xs font-medium text-slate-500 sm:text-sm">{stat.label}</div>
+          </div>
+
+          {#if i < stats.length - 1}
+            <div class="h-9 w-px bg-slate-300"></div>
+          {/if}
+        {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)